ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteJob.java

```diff
@@ -42,17 +40,14 @@ public CliCommand init(final Arguments args) throws Exception {
             throw new MissingOptionException("The delete job command requires '-i' to be set.");
         }
         this.id = UUID.fromString(args.getId());
-        this.force = args.isForce();
-        if (this.force) {
-            LOG.info("Force flag is true");
-        }
         return this;
     }
 
     @Override
     public DefaultResult call() throws Exception {
-        final CancelJobSpectraS3Request request = new CancelJobSpectraS3Request(id).withForce(this.force);
-        getClient().cancelJobSpectraS3(request);
-        return new DefaultResult("SUCCESS: Deleted job '"+ this.id.toString() +"'");
+        // Force is always on in CancelJobSpectraS3Request
```

Comment:
Aha, very good. I just did this to make it build; .withForce() was suddenly missing. No objection to fixing it here under a different issue?